Jenkinsfile

```diff
@@ -119,7 +114,7 @@ void withMavenEnv(List envVars = [], def buildType, def javaVersion, def body) {
             body.call()
         }
     }
-
+    
```

Comment:
Could we avoid the whitespace change in this line?

Reply:
it's been reverted